src/core/component.js

```diff
@@ -201,16 +201,20 @@ Component.prototype = {
       // Play the component if the entity is playing.
       this.update(oldData);
       if (el.isPlaying) { this.play(); }
+      el.emit('component-initialized', {
```

Comment:
OK. Seems like half our event names are hyphened and half are not (e.g., `enter-vr`). But I can keep it without here.